sematic/resolvers/local_resolver.py

```diff
@@ -220,9 +236,7 @@ def _resolution_did_fail(self, error: Exception) -> None:
             reason = "Marked as failed because another run in the graph failed."
             resolution_status = ResolutionStatus.COMPLETE
         else:
-            reason = (
-                "Marked as failed because of an internal error resolving the graph."
-            )
+            reason = "Marked as failed because the rest of the graph failed to resolve."
```

Comment:
So it turns out for remote jobs, we aren't able to properly distinguish failures that are from calculator errors or not here. So I changed it to a more generic error message.